lib/main.dart

```diff
@@ -185,15 +185,12 @@ class _OxCoiState extends State<OxCoi> {
               ],
               child: DynamicScreen()
             );
-
           } else if (state.configured && !state.hasAuthenticationError && !state.needsOnboarding) {
             child = Root();
-
           } else if (state.configured && state.hasAuthenticationError) {
-            child = PasswordChanged(passwordChangedCallback: () => _loginSuccess(isRelogin: true));
-
+            child = PasswordChanged(passwordChangedCallback: () => _loginSuccess);
```

Comment:
Changes the logic but should be fine, as we can assume that a password change will also invalidate all open push resources.